src/components/flowchart/styles/_layers.scss

```diff
@@ -68,3 +68,7 @@
   font-size: 1.6em;
   white-space: nowrap;
 }
+
+.pipeline-layer-name--active {
```

Comment:
We may want to add an opacity transition to the `pipeline-layer-name` base class of `350ms` or around there so it matches up more with the layer background opacity transition.

Reply:
great suggestion, added 👍